app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,36 @@
+@import 'bootstrap-sprockets';
+
+@import 'bootstrap';
+
+/* universal */
+body {
+padding-top: 60px;
```

sai indent bé ơi
check indent trong file này nhé
vd:
```suggestion
body {
  padding-top: 60px;
}
```